modules/RTC/TraceablePeerConnection.js

```diff
@@ -1145,7 +1154,7 @@ TraceablePeerConnection.prototype._processAndExtractSourceInfo = function(localS
     const media = session.media.filter(mline => mline.direction === MediaDirection.SENDONLY
         || mline.direction === MediaDirection.SENDRECV);
 
-    if (!Array.isArray(media)) {
+    if (!media?.length) {
```

Comment:
Is the optional chaining necessary? `media` will always be an array since filter always returns one.

Reply:
Yes its not, force of habit for some reason :)